advanced_source/cpp_custom_ops.rst

```diff
@@ -380,8 +469,7 @@ three ways:
 Adding training (autograd) support for an operator
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Use ``torch.library.register_autograd`` to add training support for an operator. Prefer
-this over directly using Python ``torch.autograd.Function`` or C++ ``torch::autograd::Function``;
-you must use those in a very specific way to avoid silent incorrectness (see
+this over directly using Python ``torch.autograd.Function`` (see
```

Comment:
remove reference to torch::autograd::Function since we don't support that in stable API/ABI